packages/cloud-functions/merkle-distributor-helper/package.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
To prevent these from trying to publish, you might want to add:
```
"private": true
```

to each of these package.json files.

Reply:
yup! done.